packages/plugins/telemetry/package.json

```diff
@@ -28,11 +35,7 @@
         "esbuild": "0.20.2"
     },
     "peerDependencies": {
+        "esbuild": "*",
         "webpack": "*"
```

Comment:
You should add min versions, like `"esbuild": ">=1.2.3"`

Reply:
Good call.